.pre-commit-config.yaml

```diff
@@ -38,3 +38,11 @@ repos:
     rev: v1.4.2
     hooks:
     -   id: rst-backticks
+-   repo: local
+    hooks:
+    -   id: changelogs-rst
+        name: changelog filenames
+        language: fail
+        entry: 'changelog files must be named ####.(bugfix|feature|deprecation|breaking|doc|misc).rst'
+        exclude: ^docs/changelog/(\d+\.(bugfix|feature|deprecation|breaking|doc|misc).rst|README.rst|template.jinja2)
```

Comment:
FTR chronographer was showing the red status: https://github.com/tox-dev/tox/pull/1529/checks?check_run_id=527182252. It uses the `pyproject.toml` as a source of truth for the allowed types...

Reply:
yes, but it's been pretty noisy and doesn't give people readable signal so it's unfortunately gotten ignored and slipped through -- not really your fault though, github's check results is not the most readable output :(

Reply:
Well, I mean there's an area for improvements there even on my side.